test-runner/src/main/scala/com/typesafe/spark/test/mesos/framework/runners/SparkJobRunner.scala

```diff
@@ -15,11 +18,23 @@ object SparkJobRunner {
 
     val mesosConsoleUrl = args(0)
     val deployMode = args(1)
-    val cfg = RoleConfigInfo(args(2), args(3), args(4))
-
-    val testToRun = deployMode match {
-      case "cluster" => new ClusterModeSpec(mesosConsoleUrl, cfg)
-      case "client" => new ClientModeSpec(mesosConsoleUrl, cfg)
+    val role = args(2)
+    val attributes = args(3)
+    val roleCpus = args(4)
+
+    val runFunc = () => {
```

Comment:
The test runner is now run with CLI params, as per our previous discussion and https://github.com/scalatest/scalatest/issues/848